assets/test_chambers/test_chamber_02/test_chamber_02.yaml

```diff
@@ -28,4 +35,5 @@ operators:
   - should_open_door = exit_2_0 and exit_2_1
   - should_open_door = should_open_door or bad_person
   - not_in_room = not in_room
-  - double_door = should_open_door and not_in_room
\ No newline at end of file
+  - double_door = should_open_door and not_in_room
+  - exit = exit or escape_hatch
```

Comment:
Sorry, what I meant is putting `exit` before `should_open_door`. That way all chamber 4 operators are before all chamber 5 operators, like the cutscenes, so it's easier to follow the logic.

Pushed this change just now to avoid holding up this PR for nitpicks.